src/middlewares/request-body-validation.middleware.ts

```diff
@@ -42,10 +43,13 @@ async function getValidator(req: Request) {
   }
 
   const schema = requestBody.content['application/json'].schema;
+  const serverAPIExtension = schema['x-server-api'] || {};
 
-  // asyncapi is validated in another middleware so make so annotate it as `any` type
-  if (schema.properties && schema.properties.asyncapi) {
-    schema.properties.asyncapi = true;
+  if (schema.properties) {
+    const skipFieldsInValidation: string[] = serverAPIExtension.skipFieldsInValidation || [];
+    skipFieldsInValidation.forEach(field => {
+      schema.properties[String(field)] = true;
+    });
```

Comment:
@magicmatatjahu can you kindly provide an explanation for this 😊

Reply:
We should skip validation of some particular fields, due to fact that we validate `asyncapi(s)` documents in another middleware.

Reply:
Okay got it 😅